docs/api/platform/Error.md

```diff
@@ -298,12 +329,99 @@ void save_all_errors() {
     mbed_clear_all_errors();
 }
 ```
+
+#### Using `mbed_get_reboot_error_info()` to retrieve the reboot error info
+The error context captured by the error handling system can be retrieved using mbed_get_reboot_error_info() API. See the below code for example usage of that API. 
+In the example below, a status variable reboot_error_detected has been used to track the presence of error context capture.
+
+```CPP TODO
+mbed_error_ctx error_ctx;
+int reboot_error_detected = 0;
+
+//Callback during reboot
+void mbed_error_reboot_callback(mbed_error_ctx *error_context) 
+{
+    printf("error callback received");
+    reboot_error_detected = 1;
+}
+
+// main() runs in its own thread in the OS
+int main() 
+{
+    if (reboot_error_detected == 1) {
+        if (MBED_SUCCESS == mbed_get_reboot_error_info(&error_ctx)) {
+            printf("\nSuccessfully read error context\n");
+    }
+    //main continues...
+}
+```
+
+#### Using `mbed_get_reboot_fault_context()` to retrieve the fault context info
+The fault context captured can be retrieved using mbed_get_reboot_fault_context() API. See the below code for example usage of that API. The example code below checks for error_status using the error context and then retrieves the fault context using mbed_get_reboot_fault_context() API.
```

Comment:
Same as above, open to alternative phrasing here, mainly trying to remove the API term if this sentence is only referencing a single function call (could have the usage of API in a sentence wrong as well, let me know if this is fine as is).

```suggestion
The fault context captured can be retrieved using `mbed_get_reboot_fault_context()`. See the below code for an example using that function. The example code below checks for error_status using the error context and then retrieves the fault context using `mbed_get_reboot_fault_context()`.
```